recipes/rgi/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
+cd card-data
+mv card.json ..
+cd ..
+rm -rf card-data
+./rgi load -i card.json
```

Comment:
You should be able to condense the above `cd`, `mv` statements into something like `./rgi load -i card-data/card.json`.  Also, maybe add a comment saying this is loading up the database files.